src/containers/GuestInfoContainer.tsx

```diff
@@ -0,0 +1,176 @@
+        year: date.getFullYear(),
+        month: date.getMonth() + 1, //js month starts from 0
+        day: date.getDate(),
+        hour: 0,
+        minute: 0
+      }).setZone('utc');
+      values.birthdate = new Date(formattedDate.toISO());
+
+      await axios.request(new PersonalInfoRequest(checkout.offerId, values));
+
+      dispatch({
+        type: 'SET_CHECKOUT',
+        payload: {
+          ...checkout,
+          personalInfo: values
+        }
+      });
+
+      logger.info('Guest info sent successfully');
+      navigate('/checkout/' + checkout.offerId);
+    } catch (error) {
+      const message = (error as Error).message || 'Unknown useAuthRequest error';
+      setLoading(false);
+      setError(message);
+    }
+  }, [values, checkout]);
+
+  return (
+    <FormProvider methods={methods} onSubmit={handleSubmit(onSubmit)}>
+      <Grid justifyContent="center" container spacing={3}>
+        <Grid item xs={12} md={6}>
+          <Typography>
+            This Information will be strictly used by the hotel and win.so to confirm your
+            booking! It is mandatory in order for the hotel to reserve your room.
+          </Typography>
+          <Card sx={{ my: 3, p: 3 }}>
+            <Box
+              sx={{
+                display: 'grid',
+                columnGap: 2,
+                rowGap: 3,
+                gridTemplateColumns: { xs: 'repeat(1, 1fr)', sm: 'repeat(1, 1fr)' }
+              }}
+            >
+              <RHFTextField name="firstname" label="First Name" />
+              <RHFTextField name="lastname" label="Last Name" />
+              <RHFDatePicker name="birthdate" label="Birth date" />
+              <RHFTextField name="email" label="Email Address" />
+              <RHFTextField name="phoneNumber" label="Phone Number" />
+            </Box>
+            <Stack direction="row" alignItems="flex-start" sx={{ mt: 3 }}>
+              <Checkbox checked={privacy} onChange={() => setPrivacy(!privacy)} />
+              <Box>
+                <Typography variant="subtitle1">
+                  <Link href="/privacy" underline="hover" target="_blank" rel="noopener">
+                    win.so policies
+                  </Link>
+                </Typography>
+                <Typography>
+                  Please agree to win.so general terms for travellers. Curious about how
+                  we handle your personal data? Read win.so privacy and cookie statement.
+                </Typography>
+              </Box>
+            </Stack>
+            <Stack alignItems="flex-end" sx={{ mt: 3 }}>
+              <LoadingButton
+                disabled={!privacy}
+                type="submit"
+                variant="contained"
+                loading={isSubmitting}
+              >
+                Proceed to Payment
+              </LoadingButton>
+            </Stack>
+          </Card>
+          <MessageBox loading type="info" show={loading}>
```

Comment:
We already using the `LoadingButton` to display a loading state. Is there a reason we use both?

Reply:
removed